indico/web/client/js/react/components/EditableSubmissionButton.jsx

```diff
@@ -0,0 +1,71 @@
+// This file is part of Indico.
+// Copyright (C) 2002 - 2019 CERN
+//
+// Indico is free software; you can redistribute it and/or
+// modify it under the terms of the MIT License; see the
+// LICENSE file for more details.
+
+import React, {useCallback, useState} from 'react';
+import PropTypes from 'prop-types';
+import {Button, Modal} from 'semantic-ui-react';
+import FileManager from 'indico/modules/events/editing/components/FileManager';
+import {fileTypePropTypes} from 'indico/modules/events/editing/components/FileManager/util';
+import {indicoAxios, handleAxiosError} from 'indico/utils/axios';
+
+export default function EditableSubmissionButton({
+  eventId,
+  contributionId,
+  uploadURL,
+  fileTypes,
+  type,
+  submitRevisionURL,
+}) {
+  const [submissionFiles, setSubmissionFiles] = useState({});
+  const [submitting, setSubmitting] = useState(false);
+  async function submitRevision() {
+    setSubmitting(true);
+    try {
+      await indicoAxios.put(submitRevisionURL, {
+        files: submissionFiles,
+      });
+    } catch (e) {
+      handleAxiosError(e, false, true);
+      setSubmitting(false);
+      return;
+    }
+    setSubmitting(false);
+    window.location.href = `/event/${eventId}/contributions/${contributionId}/editing/${type}`;
```

Comment:
import `indico-url:event_editing.editable` and use that to build the url. hardcoding doesn't only break in case we ever change the url blueprint, but also when running indico in a subdir (even though we don't really support that anymore)